x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_dashboard_link.tsx

```diff
@@ -34,10 +34,14 @@ function useAgentDashboardLink(agent: Agent) {
 
 export const AgentDashboardLink: React.FunctionComponent<{
   agent: Agent;
-}> = ({ agent }) => {
+  agentPolicy?: AgentPolicy;
+}> = ({ agent, agentPolicy }) => {
```

Comment:
nit: it would be enough to just pass the `monitoring_enabled` property instead of the `agentPolicy`

Reply:
yes it would be enough but not sure it make the component easier to use or it will provide other value